lib/polymorphic_integer_type/extensions.rb

```diff
@@ -95,7 +95,7 @@ def remove_type_and_establish_mapping(name, options, scope)
       def retrieve_polymorphic_type_mapping(polymorphic_type:, class_name:)
         return if polymorphic_type.nil?
 
-        belongs_to_class = class_name.safe_constantize
+        belongs_to_class = compute_type(class_name)
```

Comment:
`safe_constantize` returns `nil` when it can't find a constant, but `compute_type` raises a `NameError`. I suppose this exception would happen at some point in the process of setting up/accessing an association, but are there any other assumptions in our extension that need to change as a result?

Reply:
I can't think of a situation where it would return `nil`, and this gets called during class loading so I think if things work in an eager loaded environment, we should be good.